src/pages/Trainee/TraineeList.jsx

```diff
@@ -57,16 +96,41 @@ const TraineeList = (props) => {
             format: getDateFormatted,
           },
         ]}
+        actions={[
+          {
+            icon: <EditIcon />,
+            handler: handleEditDialogOpen,
+          },
+          {
+            icon: <DeleteIcon />,
+            handler: handleDeleteDialogOpen,
+          },
+        ]}
         order={order}
         orderBy={orderBy}
         onSort={handleSort}
         onSelect={handleSelect}
+        page={page}
+        onChangePage={handleChangePage}
+        count={100}
+        rowsPerPage={5}
```

Comment:
why use har-coded props value for the count and rowsPerPage ?
Use states  and pass them